src/resolve/MaxSum.jl

```diff
@@ -1,15 +1,5 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
```

Comment:
This is supposed to be lower case but I can't change that because of maclol....???